app/models/manageiq/providers/workflows/automation_manager/workflow_instance.rb

```diff
@@ -49,13 +49,15 @@ def queue_callback(state, message, _result)
   end
 
   def run(args = {})
-    zone, role, object_type, object_id = args.values_at(:zone, :role, :object_type, :object_id)
+    queue_args = args.slice(:zone, :role, :object_type, :object_id)
+    zone, role, object_type, object_id = queue_args.values_at(:zone, :role, :object_type, :object_id)
+
+    ManageIQ::Providers::Workflows::Runner.runner.add_workflow(self, queue_args)
```

Comment:
note to self/please verify:
This is recorded so when an event comes from docker, it can be put back into the miq_queue.

- Are these zone/role values going to be the same for all messages?
- Is object_type/object_id pretty the workflow id?

Reply:
> Are these zone/role values going to be the same for all messages?

Currently role will always be automate, I have these in case we wanted to do e.g. builtin workflows for ems_operations like we do with ansible_runner.

Zone can be different.

> Is object_type/object_id pretty the workflow id?

No object_type/object_id is typically a `ServiceTemplateProvisionTask` from a Service Order

Reply:
These queue args are not the same for all workflow_instances which is why we are keeping the queue args with with specific workflow_instance.